main.go

```diff
@@ -64,7 +64,7 @@ func init() {
 	_ = scheme.AddToScheme(myscheme)
 	_ = infrav1.AddToScheme(myscheme)
 	_ = clusterv1.AddToScheme(myscheme)
-	_ = kubevirtv1.AddToScheme(myscheme)
+	_ = kubevirtscheme.AddToScheme(myscheme)
```

Comment:
If you stick with the dynamic client,I think you can completely avoid generating the client if you do smoething like this:


```
_ = kubevirtv1.AddToScheme(myscheme)
```

since `kubevirt.io/api` already comes with `AddToScheme` functions. I see that the generated client seems to create the scheme slightly different, so something to try out.

Reply:
Done